content/departments/engineering/ship/aws.md

```diff
@@ -66,3 +75,13 @@ You can only delete AWS accounts which belong to your team. To delete an AWS acc
 2. Open Pull Request and ask for approval in #discuss-cloud-ops channel.
 
 3. [For Cloud Operations] After merging PR, follow [instructions](https://sourcegraph.sourcegraph.com/github.com/sourcegraph/infrastructure/-/blob/cloud/aws/README.md#create-new-aws-account)
+
+## FAQ
+
+### How can I use AWS IAM Users credentials for automation?
+
+We strongly recommend using AWS SSO role instead of using IAM user credentials for automation. However if you need IAM user credentials for any reason, they are only allowed in non-production accounts.
```

Comment:
we should say it's mandatory

Reply:
For non-production accounts, I think this is still fine to experiment with AWS IAM credentials (most tutorials show it this way, so we do not want to people to waste time learning the alternative ways of access to make quick spike imho), but it's explicitly said that `only allowed in non-production` (so in test accounts).